docs/src/examples/components/Flex/Types/FlexExampleColumns.shorthand.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+import { Flex, Segment } from '@stardust-ui/react'
+
+const FlexExampleColumns = () => (
+  <>
+    <Flex gap="gap.small" padding="padding.medium">
+      <Flex.Item size="size.half">
+        <Segment content="1/2" />
+      </Flex.Item>
+
+      <Flex.Item size="size.half">
+        <Segment content="1/2" />
+      </Flex.Item>
+    </Flex>
+
+    <Flex gap="gap.small" padding="padding.medium">
+      <Flex.Item size="size.quarter">
+        <Segment content="1/4" />
+      </Flex.Item>
+
+      <Flex.Item size="size.half">
+        <Segment content="1/2" />
+      </Flex.Item>
+
+      <Flex.Item size="size.quarter">
+        <Segment content="1/4" />
+      </Flex.Item>
+    </Flex>
+
+    <Flex gap="gap.small" padding="padding.medium" style={{ minHeight: 200 }}>
```

Comment:
what about having `height` as prop? I imagine it was considered already? Disregard in that case

Reply:
yes, actually, it was - and the main reason for not doing it (for now, at least) is that `height` wasn't seen as a close attribute of the layout aspects - in other case, this prop is not specific to layout component (the same way then we might argue about adding `height` to API of Segment). 

Also, note that if Flex needs to be provided with some height while being part of some other component, this will be done by styles, not by the props API:

```jsx
renderComponent = ({ ..., ElementType, styles }) => {
   <ElementType>
       <Flex styles={styles.flex} />
  </ElementType>
}

/// myComponentStyles.js
{
  root: ...
  flex: {
    height: '200px'
  }
}
```